lib/trento/application/integration/discovery/payloads/cluster/crmmon_discovery_payload.ex

```diff
@@ -213,6 +221,12 @@ defmodule Trento.Integration.Discovery.ClusterDiscoveryPayload.Crmmon do
     |> validate_required_fields(@required_fields)
   end
 
+  defp nodes_changeset(nodes, attrs) do
+    nodes
+    |> cast(attrs, [:id, :name, :online, :unclean])
+    |> validate_required([:id, :name])
```

Comment:
This is `validate_required` and not `validate_required_fields` in purpose, as the 2nd doesn't work properly in this case, as it filters out :id` as the main schema doesn't have this field